docs/articles/infrastructure/ci-cd/considerations.rst

```diff
@@ -0,0 +1,23 @@
+:orphan:
+
+Considerations for Continuous Integration jobs
+==============================================
+
+General strategy
+----------------
+Continuous integration and continuous delivery jobs serve the purpose of quick
+turn-around time for developers, providing easy feedback on work in a controlled
+environment. As such, our jobs are designed with some basic properties:
+
+* Build on-commit
+* Fail builds early
+* Make build status visible
+* Make sure latest build is available to users
```

Comment:
Build or build artifacts? What about nightly builds?

Reply:
This talks about the broader sense where the build could be a deployed web page or a full yocto build or anything really. The latest should be readily available anyhow.